workspace-server/src/main/java/gov/nasa/jpl/aerie/workspace/server/WorkspaceBindings.java

```diff
@@ -144,6 +172,37 @@ private JWTService.UserSession authorize(Context context) {
     }
   }
 
+  /**
+   * Check that the request meets the permissions to perform the given action on the workspace.
+   * If it does not, format the context into an appropriate error state.
+   * @return true, if the user passes the permissions check. false otherwise
+   */
+  private boolean checkPermissions(Context context, int workspaceId, WorkspaceAction action) {
```

Comment:
No action needed, but i wonder if a better pattern here could be to `throw` on failed permission checks rather than mutating context and returning a boolean. That would centralize error handling, keep this helper more pure, and avoid callers needing to remember to `return` after the `false` case.